src/CommunityToolkit.Maui.Core/Platform/KeyboardManager/KeyboardExtensions.net.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Diagnostics;
+using System.Reflection.Metadata;
+
+namespace CommunityToolkit.Maui.Core.Platform;
+
+public static partial class KeyboardExtensions
+{
+	static bool HideKeyboard(this object _)
+	{
+		return false;
```

Comment:
Should we return false or throw exception similar to macOS?

Reply:
Good catch. Let's throw a `NotSupportedException`